tox.ini

```diff
@@ -49,9 +56,9 @@ exclude =
 
 [testenv:lint]
```

Comment:
Just above here, could we add `.eggs/*,` to the exclude section of the [flake] settings? 
when I run `tox -e lint` it's linting up files in that directory.